docs/user_manual/processing_algs/qgis/vectoroverlay.rst

```diff
@@ -1048,11 +1048,11 @@ attribute values from both layers for overlapping features.
 .. I'm not sure if the note below is relevant/true with the multiunion alg
 
    .. note::
-   For ``union(A,B)`` algorithm, if there are overlaps among
+   For ``union(A,B,...)`` algorithm, if there are overlaps among
    geometries of layer A or among geometries of layer B, these are not
-   resolved: you need to do ``union(union(A,B))`` to resolve all
+   resolved: you need to do ``union(union(A,B,...))`` to resolve all
```

Comment:
> maybe should all occurrences of union in this note become also multiunion here

Makes sense to me. And I just checked and it does work.

Reply:
Honestly this note is very heavy on the math/programming language. It works for me but I can see my students eyes glazing over instantly. 
More plain language would be: Features on the same layer will not split each other. If you want to split overlaps on the same layer as well as other layers first run the algorithm with multiple layers then run the algorithm again with the multi-layer output as the only input.

Reply:
🤣 but You are right. Trying to avoid input and output too close, u OK with
`
Features on the same layer will not split each other. If you want to split overlaps on the same layer as well as other layers, first run the algorithm with multiple layers then run the algorithm again with only the previous output.`?

Reply:
Looks good to me!

Reply:
done in [9f90d15](https://github.com/qgis/QGIS-Documentation/pull/7613/commits/9f90d15e78de511b84039a06902967975c001d1a) with a tweak since it needs some context.